### PR TITLE
Updated Spanish translation

### PR DESCRIPTION
**Pull request by husslede**
# Description

Added new Spanish translations

## Type of change

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

## Limitations

none

### CODE REVIEW

File: src/client/app/translations/data.json [hunk 1]
```diff
@@ -656,11 +656,11 @@
 		"map.load.complete": "Carga de mapas completa de",
 		"map.modified.date": "Ultima modificación",
 		"map.name": "Nombre de mapa",
-		"map.new.angle": "(Need Spanish) List the angle with respect to true north (0 to 360)",
+		"map.new.angle": " Enumere el ángulo con respecto al norte verdadero (0 a 360)",
 		"map.new.name": "Defina un nombre para el mapa:",
 		"map.new.submit": "Guardar y continuar",
 		"map.new.upload": "Subir la imagen del mapa para empezar.",
-		"map.notify.calibration.needed": "(Need Spanish) Calibration needed before display", 
+		"map.notify.calibration.needed": "Nececitar calibración antes de visualizar
```
Comment (huss) [inline src/client/app/translations/data.json]: Missing end quote and comma.

File: src/client/app/translations/data.json [hunk 2]
```diff
@@ -584,9 +584,9 @@
 		"failed.to.submit.changes": "No se pudo entregar los cambios",
 		"graph.type": "Tipo de gráfico",
 		"group": "Grupo",
-		"group.gps": "(Need Spanish) Group GPS (latitude, longitude)",
-		"group.gps.error": "(Need Spanish) Please input a valid GPS: (latitude, longitude)",
-		"group.id": "(Need Spanish) Group ID",
+		"group.gps": "Grupo de GPS: (latitud, longitud)",
```
Comment (huss) [inline src/client/app/translations/data.json]: This translation is fine but it is a group's GPS in this case so maybe GPS de grupo?

File: src/client/app/translations/data.json [hunk 3]
```diff
@@ -638,12 +638,12 @@
 		"manage": "Gestionar",
 		"map": "Mapa",
 		"maps": "Mapas",
-		"map.bad.number": "(Need Spanish) Not a number, please change angle to a number between 0 and 360",
-		"map.bad.digita": "(Need Spanish) Greater than 360, please change angle to a number between 0 and 360",
-		"map.bad.digitb": "(Need Spanish) Less than 0, please change angle to a number between 0 and 360",
+		"map.bad.number": "Ni un número, por favor cambiar el angúlo entre 0 a 360",
+		"map.bad.digita": "Mayor que 360, por favor cambiar el angúlo a un número entre 0 a 360",
+		"map.bad.digitb": "Menor que 360, por favor cambiar el angúlo a un número entre 0 a 360",
```
Comment (huss) [inline src/client/app/translations/data.json]: I think this should be Menor que 0 not 360.

**Review by huss — CHANGES_REQUESTED** [review]
Thank you for doing this. A few small items to address.

File: src/client/app/translations/data.json (hunk 1 above)
Comment (husslede) [reply src/client/app/translations/data.json]: fixed

File: src/client/app/translations/data.json (hunk 2 above)
Comment (husslede) [reply src/client/app/translations/data.json]: fixed

File: src/client/app/translations/data.json (hunk 3 above)
Comment (husslede) [reply src/client/app/translations/data.json]: thank you for finding that - fixed

**Review by huss — APPROVED** [review]
Thank you to @husslede for continuing to work on Spanish translations. This is now ready to merge.

**Pull request merged**